src/UBAFeeCalculator/UBAFeeUtility.ts

```diff
@@ -292,56 +290,41 @@ export function computePiecewiseLinearFunction(
   return integral.mul(scale);
 }
 
-/**
- * Computes the utilization at a given point in time based on the
- * current balances and equity of the hub and spoke pool targets.
- * @param decimals The number of decimals for the token
- * @param hubBalance The current balance of the hub pool for the token
- * @param hubEquity The current equity of the hub pool for the token
- * @param ethSpokeBalance The current balance of the ETH spoke pool for the token
- * @param targetSpoke The current balance of the target spoke pool for the token - this is a list.
- * @returns The utilization of the hub pool
- */
 export function calculateUtilization(
   decimals: number,
   hubBalance: BigNumber,
-  hubEquity: BigNumber,
-  ethSpokeBalance: BigNumber,
-  spokeTargets: { target: BigNumber; spokeChainId: number }[],
-  hubPoolChainId = HUBPOOL_CHAIN_ID
+  hubLiquidReserves: BigNumber,
+  ethSpokeDelta: BigNumber,
+  cumulativeSpokeTargets: BigNumber
 ) {
-  const numerator = hubBalance
-    .add(ethSpokeBalance)
-    .add(spokeTargets.reduce((a, b) => (b.spokeChainId !== hubPoolChainId ? a.add(b.target) : a), BigNumber.from(0)));
-  const denominator = hubEquity;
+  const numerator = hubLiquidReserves.add(ethSpokeDelta).add(cumulativeSpokeTargets);
+  const denominator = hubBalance;
```

Comment:
Would we be susceptible to the same vector as we were discussing in the call today? The case where a user sends funds to the hub pool contract directly